test/experimental_apis/fetch.js

```diff
@@ -54,11 +54,51 @@ describe('Experimental fetch function', () => {
     return should(networkResource.content.get(filePath)).be.fulfilled();
   });
 
-  // not supported yet, it needs support for XOR-URL
-  it.skip('fetch a MD resource', async () => {
+  it('fetch a MutableData resource', async () => {
+    const content = `hello world, on ${Math.round(Math.random() * 100000)}`;
+    const { serviceMd } = await createRandomDomain(content, '', '', app);
+    const info = await serviceMd.getNameAndTag();
+    should(info.xorUrl).not.be.undefined();
+    const networkResource = await unregisteredApp.fetch(info.xorUrl);
+    should(networkResource.resourceType).equal('MD');
```

Comment:
I think we `should(networkResource.content).equal( content )` or some such, to ensure we're grabbing the correct resource

Reply:
Let me see what I can do, but the content is the actual MutableData object, so perhaps I'll have to iterate thru the entries to see if they contain what were supposed to